acceptance/framework/k8s/debug.go

```diff
@@ -149,6 +149,19 @@ func WritePodsDebugInfoIfFailed(t *testing.T, kubectlOptions *k8s.KubectlOptions
 				writeResourceInfoToFile(t, service.Name, "service", testDebugDirectory, kubectlOptions)
 			}
 		}
+
+		// Get the Events from Kubernetes, this helps track down scheduling issues.
```

Comment:
This was just for debugging purposes and can be backed out later, its a bit verbose.